content/slides/data-formats-and-inspection.md

```diff
@@ -21,114 +21,45 @@ The best formats for storing geospatial data have been designed for this purpose
 Some general-purpose formats like CSV may be okay, but not great.
 :::
 
-
 ## Finding information about formats
 
 * [Library of Congress list of digital
 formats](https://www.loc.gov/preservation/digital/formats/fdd/browse_list.shtml)
     * Check the "Licensing and patents" section to learn about openness of a format.
 
 * [Open Geospatial Consortium (OGC)-adopted standards](https://www.ogc.org/standards/)
-    * In these slides, 🌎 indicates an OGC-adopted standard.
+  * In these slides, 🌎 indicates an [OGC-adopted
+    standard](https://www.ogc.org/standards/).
+    
+* [More slides on common data formats](/content/slides/more-slides-on-data-formats.html)!
```

Comment:
I'd like to come up with a more succinct naming convention that will keep the related slide decks together when alphabetically organized. What do you think of `data-formats-and-inspection-more.md`, `-extra`, or `-bonus`?